api/config/data_feed/config_request.proto

```diff
@@ -11,10 +11,13 @@ import "google/protobuf/wrappers.proto";
 message ConfigRequest {
 	option (chef.automate.api.service_config) = {name: "data-feed-service"};
 
-	V1 v1 = 1;
+    reserved 1 to 2;
+
+	V1 v1 = 3;
```

Comment:
If you do this, people will lose any configuration they've set on your service.

Reply:
both these lines?

```
    reserved 1 to 2;

	V1 v1 = 3;
```

Reply:
If you change the field id for `v1`, it means previously saved protos will no longer populate the v1 field.  You can remove `reserved 1 to 2;` and keep `V1 v1 = 1;` as before.